### ISSUE TITLE
Flowershow website - improvements Jan 2023

### ISSUE THREAD

**Issue by rufuspollock** [datopian/flowershow#410]
Variety of site improvements as of Jan 2023

## Tasks

- [x] #247
- [ ] review our home page and see what can be improved
- [ ] review our docs and update them if necessary
- [x] remove this text, this is not needed for regular users
   ![image](https://user-images.githubusercontent.com/52197250/219712063-f1387d1e-9a98-463e-b002-d79f76120fd1.png)
- [x] #455
- [x] make this link point directly to our gh roadmap board
   ![image](https://user-images.githubusercontent.com/52197250/219715823-0d71f9d7-c72f-467d-a84e-1c2beeb13ad7.png)
- [x] hide "Created by Flowershow", it's obvious and looks awful (should be easy to add a bool prop in config; I bet not everyone wants to show it in their footer anyway)
   ![image](https://user-images.githubusercontent.com/52197250/219717007-69a1ae98-e4ed-4815-bb88-4ab50104f5bf.png)
- [x] make sure all our docs pages and tutorials are listed in /docs/index.md
  - [x] also (if needed): adjust their titles and names used in links on the /docs/index.md page
- [x] remove this part from `/roadmap` page (?)
   ![image](https://user-images.githubusercontent.com/52197250/219721087-1c8edb2c-3499-4e2e-bd8f-49d24d914b62.png)
- [x] investigate and fix broken (or outdated ?) Algolia results **✅2023-12-19 ❌ couldn't fix and moved to own issue #609**

**Comment by olayway** [comment]
@PhilippeduPreez I've added some tasks to this issue. If you have time this sprint, maybe you could at least start fixing some of these?

**Comment by olayway** [comment]
@rufuspollock we'll need access to algolia portal to investigate the search issues

**Comment by PhilippeduPreez** [comment]
@olayway I think the  hover-over tooltips can only be enabled or disabled for all pages and not specifically the home page, and on other index pages at the moment. So should I disable it for all with:
```
const config = {
  showLinkPreviews: false,
};
```
like shown at https://flowershow.app/docs/link-previews

**Comment by khalilcodes** [comment]
@PhilippeduPreez @olayway This is a quick fix if needed.

**Comment by PhilippeduPreez** [comment]
@khalilcodes I'll add the hover-over functionality change in a new issue and make you reviewer for the PR

**Comment by olayway** [comment]
> @olayway I think the hover-over tooltips can only be enabled or disabled for all pages and not specifically the home page, and on other index pages at the moment. So should I disable it for all with:

Yes @PhilippeduPreez, I know, and this is what I meant.

**Comment by PhilippeduPreez** [comment]
@rufuspollock @khalilcodes Or if we have Crawler Public API access, we could implement a github action https://github.com/algolia/algoliasearch-crawler-github-actions

**Comment by rufuspollock** [comment]
> @rufuspollock @khalilcodes Can anyone grant me access to our Algolia account? I think the search problem is due to not triggering a reindexing.

@PhilippeduPreez i've investigated now and i think only i can have access (no way to add others) - and it is tied to my github account.

I found the issue which was that crawl had stalled  for some reason (many months ago - see date in screenshot below which is last august!). I've now restarted it so re-indexing should happen soon.

<img width="1108" alt="image" src="https://user-images.githubusercontent.com/180658/223746281-40649e8d-bd5a-46d2-bd9a-08230f63aa13.png">
